tests/test_contexts.py

```diff
@@ -189,3 +189,14 @@ def test_sim_context():
 def test_offline():
     st = xenonnt('latest')
     st.provided_dtypes()
+
+
+@unittest.skipIf(not straxen.utilix_is_configured(), "No db access, cannot test!")
+def test_xedocs_global_version_hash_coinsistency():
+    st1 = xenonnt_online()
+    st2 = xenonnt_online()
+    # Same URLs but the quesries are in different order
+    st1.apply_xedocs_configs(version = 'v_test1', db='development_db')
```

Comment:
Tests should not depend on the state of the development database. 
Please set the context config in the test using two differently ordered URLs.
Move the test to test_url_config.py